notebooks/book1/01/emnist_viz_jax.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "7351151b",
    "metadata": {},
    "source": [
-    "Please find torch implementation of this notebook here: https://colab.research.google.com/github/probml/pyprobml/blob/master/notebooks/book1/01/emnist_viz_torch.ipynb"
+    "Please find an alternative PyTorch implementation of this notebook here: https://colab.research.google.com/github/probml/pyprobml/blob/master/notebooks/book1/01/emnist_viz_torch.ipynb"
```

Comment:
Can you do this for all d2l notebooks in a separate PR? because this affects all d2l notebooks too. For now, I am merging this change for this particular notebook.